client/js/store/modules/auth.js

```diff
@@ -107,5 +106,47 @@ export default {
         );
       });
     },
+    requestStatus(context) {
+      return new Promise((resolve, reject) => {
+        // In a development scenarios, we need to ping the server to get the cookie on
+        // the vue frontend
+        if (process.env.NODE_ENV === 'development') {
+          fetch('/user/ping', {
+            credentials: 'include',
+          });
+        }
+        io.socket.get('/user/status', {}, async function handleResponse(resData, jwres) {
```

Comment:
```suggestion
        io.socket.get('/user/status', {}, function handleResponse(resData, jwres) {
```
Need to remove this if we're also removing the below await.